packages/account/src/providers/transaction-request/transaction-request.ts

```diff
@@ -615,12 +609,12 @@ export abstract class BaseTransactionRequest implements BaseTransactionRequestLi
       const assetInput = findAssetInput(assetId);
 
       if (assetInput && 'assetId' in assetInput) {
-        assetInput.id = generateId();
+        assetInput.id = hexlify(randomBytes(33));
```

Comment:
What's the significance of `33` here? Seems like a magic number

Reply:
Type of `32`? 🤔

Reply:
If it is, I think that further consolidates the notion that we should move away from [magic numbers](https://en.wikipedia.org/wiki/Magic_number_(programming) in our code as much as possible.

Reply:
UTXOs IDs have 33 bytes:
- The first 32 bytes (`b256`) represent the ID of the transaction that created the UTXO.
- The last part, 1 byte (`u8`), indicates the output index where the corresponding [OutputCoin](https://specs.fuel.network/master/tx-format/output.html#outputcoin) was located within the Outputs of the originating Transaction. ( on VM version 0.48.0, this [changed](https://github.com/FuelLabs/fuels-ts/pull/2021/commits/a94624ddae4e8c74527fb4d723e3c554c5b7627f) to 2 bytes (`u16`) )

I can create a constant with this value to be used instead of the number.

Reply:
Thanks for clarifying. Yes I agree would be good to name those constants.

Reply:
https://github.com/FuelLabs/fuels-ts/pull/2034/commits/7fc0e27d3f5bc103001248f7f2dbc8211c16f00c